modules/openapi-generator/src/main/resources/kotlin-spring/libraries/spring-boot/buildGradle-sb3-Kts.mustache

```diff
@@ -0,0 +1,57 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "{{groupId}}"
+version = "{{artifactVersion}}"
+java.sourceCompatibility = JavaVersion.VERSION_17
+
+repositories {
+    mavenCentral()
+    maven { url = uri("https://repo.spring.io/milestone") }
+}
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions.jvmTarget = "17"
+}
+
+plugins {
+    val kotlinVersion = "1.7.10"
+    id("org.jetbrains.kotlin.jvm") version kotlinVersion
+    id("org.jetbrains.kotlin.plugin.jpa") version kotlinVersion
+    id("org.jetbrains.kotlin.plugin.spring") version kotlinVersion
+    id("org.springframework.boot") version "3.0.0-M5"
+    id("io.spring.dependency-management") version "1.0.14.RELEASE"
+}
+
+dependencies {
+{{#reactive}}    val kotlinxCoroutinesVersion = "1.6.1"
+{{/reactive}}    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
```

Comment:
Nice to have implementation here, instead of compile like for the default template. compile doesn't exist any more for Gradle 7

Reply:
Should I change something?

Reply:
Nope, just praising you!

Reply:
kudos to @jayandran-Sampath